src/main/resources/view/MainWindow.fxml

```diff
@@ -65,13 +65,6 @@
           </padding>
         </StackPane>
 
-<!--        <VBox fx:id="entryList" minWidth="340" prefWidth="340" styleClass="pane-with-border" VBox.vgrow="ALWAYS">-->
-<!--          <padding>-->
-<!--            <Insets bottom="10" left="10" right="10" top="10" />-->
-<!--          </padding>-->
-<!--          <StackPane fx:id="entryListPanelPlaceholder" VBox.vgrow="ALWAYS" />-->
-<!--        </VBox>-->
```

Comment:
Is this commented out? If so can just remove!

Reply:
Yup removed in a newer commit!